src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -190,7 +190,7 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
           : html`
               <paper-input
                 id="alarmCode"
-                label="Alarm Code"
+                label="[[localize('ui.card.alarm_control_panel.alarm_code')]]"
```

Comment:
This is the way it is done in PolymerElements, this is a LitElement. This won't work.

`.label=${this.hass.localize(...)}`

Also, please test your code before making a PR.

Reply:
Fixed. I hope it's ok now.

Reply:
I had to hacked the code to test it with any entity because I don't have any working alarm integration on my localhost...

Reply:
You can always use the demo integration:

```
demo:
```

Reply:
Thanks.